### ISSUE TITLE
gpg: WARNING: "--no-show-photos" is a deprecated option

### ISSUE THREAD

**Issue by fergbrain** [zertrin/duplicity-backup#180]
```
gpg: WARNING: "--no-show-photos" is a deprecated option
gpg: please use "--list-options no-show-photos" instead
```

**Comment by zertrin** [comment]
Hi, thanks for reporting this.

Could you check what is the first version of gpg which accept the new format? 

If it is too recent an addition, we need to handle the backward compatibility with a version check at runtime...

**Comment by fergbrain** [comment]
The current format was introduced in 1.3.3 (late-2003)[0]:

> * New --list-options option.  This option takes a list of
>       arguments that allows the user to customize exactly what key
>       listings (including the --edit-key listing) look like, enabling
>       or disabling things such as photo display, policy URL, preferred
>       keyserver URL, or notation display, long or short keyIDs,
>       calculated validity for each user ID, etc.  See the manual for
>       the complete list of list-options.


It appears the old version became fully depreciated somewhere around 1.4.3 (mid-2006) [1].

I'm not aware of any sane distributions that still support that version. For example, Debian 6 shipped with version 1.4.12 [2] and LTS for that ended in 2016[3].

[0] https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=blob_plain;f=NEWS;hb=refs/heads/STABLE-BRANCH-1-4
[1] https://dev.gnupg.org/rG29b23dea9731e8f258211bc6fd733d205c18e2a8
[2] https://packages.debian.org/wheezy/gnupg
[3] https://wiki.debian.org/LTS

**Comment by zertrin** [comment]
Thank you very much for the detailed references, and sorry for the delay on my side. Terribly busy week is now behind me.

**Comment by zertrin** [comment]
closed by #181
